react/react1/week1/todo-list/pages/index.js

```diff
@@ -0,0 +1,64 @@
+import { Inter } from 'next/font/google'
+
+import { useEffect, useState } from 'react'
+
+import Title from '@/components/Title'
+import Footer from '@/components/Footer'
+import SearchTodo from '@/components/SearchTodo'
+import TodoList from '@/components/TodoList'
+import AddTodo from '@/components/AddTodo'
+
+import styles from '@/styles/Home.module.css'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export default function Home() {
+  const [todos, setTodos] = useState(['get out of the bed', 'brush teeth', 'eat breakfast'])
```

Comment:
I would recommend putting the "todo" items in a separate .json file (or you can create an api for storing them, if you want some practice)